test/kibana_dashboards.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Test script that loads dashboards
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/deployer/scripts/util.sh"
+trap os::test::junit::reconcile_output EXIT
+os::util::environment::use_sudo
+
+os::test::junit::declare_suite_start "test/kibana_dashboards"
+
+if [ -n "${DEBUG:-}" ] ; then
+    set -x
+    curl_output() {
+        python -mjson.tool
+    }
+else
+    curl_output() {
+        cat > /dev/null 2>&1
+    }
+fi
+
+espod=$( oc get pods --selector component=es -o jsonpath='{ .items[*].metadata.name }' )
+esopspod=$( oc get pods --selector component=es-ops -o jsonpath='{ .items[*].metadata.name }' )
+esopspod=${esopspod:-$espod}
+
+LOG_ADMIN_USER=${LOG_ADMIN_USER:-admin}
+LOG_ADMIN_PW=${LOG_ADMIN_PW:-admin}
+
+if oc get users "$LOG_ADMIN_USER" > /dev/null 2>&1 ; then
+    os::cmd::debug Using existing user $LOG_ADMIN_USER
+else
+    os::cmd::info Creating cluster-admin user $LOG_ADMIN_USER
+    current_project="$( oc project -q )"
+    os::cmd::debug "$( oc login --username=$LOG_ADMIN_USER --password=$LOG_ADMIN_PW )"
```

Comment:
I don't believe this is necessary since when adding roles to a user, a user doesn't need to exist.  I'm pretty certain the association is by name and userpolicy is applied to users with that name.  There is no 'user' store per se.